playbooks/plantain_staging.yml

```diff
@@ -10,6 +10,7 @@
     - role: roles/pulibrary.ruby
     - role: roles/pulibrary.postgresql
     - role: roles/pulibrary.openjdk
+    - role: roles/pulibrary.subversion-pulfa
```

Comment:
is this role a dependency of plantain? if yes why not add this in the `meta/main.yml` of the `pulibrary.plantain` role to prevent it running two times?